modules/data.atmosphere/R/met.process.R

```diff
@@ -12,21 +12,14 @@
 ##' @param dir  directory to write outputs to
 ##' 
 ##' @author Elizabeth Cowdery, Michael Dietze
-met.process <- function(site, input_met, start_date, end_date, model, host, bety, dir, browndog=NULL){
+met.process <- function(site, input_met, start_date, end_date, model, host, dbparms, dir, browndog=NULL){
   
   require(RPostgreSQL)
```

Comment:
So is the library RPostgreSQL supposed to be loaded in db.open?
I tried running db.open(dbparms) without having loaded RPostgreSQL and got the error message "... could not find function 'PostgreSQL' "